_scalingo_api/events.md

```diff
@@ -1211,11 +1211,19 @@ _When:_ Each time an SCM integration link is created to an app
 `type=link_scm`
 
 {:.table}
-| field           | type   | description                                 |
-| --------------- | ------ | ------------------------------------------- |
-| repo_name       | string | Name of the repository                      |
-| linker_username | string | Username of the linker                      |
-| source          | string | Source URL of the repository                |
+| field                        | type    | description                                                                                          |
+| ---------------------------- | ------- | ---------------------------------------------------------------------------------------------------- |
+| repo_name                    | string  | Name of the repository                                                                               |
+| linker_username              | string  | Username of the linker                                                                               |
+| source                       | string  | Source URL of the repository                                                                         |
+| branch                       | string  | Branch on which the repository is tied                                                               |
```

Comment:
In English, ties are "to" something, not "on".

You can either write "Branch to which the repository is tied", or "Branch of the repository the integration link is made to".

Reply:
You're right, fixing it